packages/web/src/components/comments/CommentBlock.tsx

```diff
@@ -57,12 +57,27 @@ const CommentBlockInternal = (
     (state: AppState) => getUser(state, { id: commentUserId })?.handle
   )
 
-  const { artistId } = useCurrentCommentSection()
-
-  const [deleteComment, { status: deleteStatus }] = useDeleteComment()
-  const [, { status: commentPostStatus }] = usePostComment() // Note: comment post status is shared across all inputs they may have open
-  const isDeleting = deleteStatus === Status.LOADING
+  const { artistId, entityId } = useCurrentCommentSection()
+
+  const [deleteComment] = useDeleteComment()
+
+  // TODO: whats a better way to package this?
+  // Need to get the status of this comment regardless of where the usePostComment hook was called
```

Comment:
Could we do `usePostCommentStatus(id)` or something?

Reply:
hmm yeah I'll set that up. Gonna merge this for now and continue on the mobile side